src/style.css

```diff
@@ -1,8 +1,92 @@
+/* ROOT ELEMENTS */
+
 #app {
   font-family: Avenir, Helvetica, Arial, sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   text-align: center;
   color: #2c3e50;
-  margin-top: 60px;
+  margin-top: 0px;
+}
+
+:root {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
```

Comment:
i think those *::before and *::after can be simplify to ::before and ::after, sorry my bad. check that out